docs/porting/libraries.md

```diff
@@ -1,5 +1,303 @@
+
+## .NET Core project model overview
+
+.NET Core introduces a new project model that you may wish to spend some time understanding before you begin porting your code.  Rather than provide a comparison with the .NET Framework project model, this section covers the essential pieces.
+
+### The project file: `project.json`
+
+.NET Core projects are defined by a directory containing a `project.json` file.  This file is where all aspects of the project are declared, including package dependencies, compiler configuration, runtime configuration, and more.
+
+The `dotnet restore` command reads this project file, restores all dependencies of the project, and generates a `project.lock.json` file.  This file contains all the necessary information the build system needs to build the project.
+
+To learn more about the `project.json` file, read the .NET Core project system docs. **NOTE: need link here to the official docs we're generating**.
+
+### The solution file: `global.json`
+
+The `global.json` file is an optional file to include in a solution which contains multiple projects.  It typically resides in the root directory of a set of projects.  It can be used to inform the build system of different subdirectories which can contain projects.  This is for larger systems composed of many projects.
+
+To learn more about the `global.json` file, check out the .NET Core project system docs.
+
+## Targeting your code for .NET Framework 4.6.1
+
+If your code is not targeting .NET Framework 4.6.1, it's highly recommended that you retarget.  This ensures that you can use API alternatives for cases where .NET Core can't support existing APIs.
+
+For each of your projects in Visual Studio you wish to port, do the following:
+
+1. Right-click on the project and select Properties
+2. In the "Target Framework" dropdown, select ".NET Framework 4.6.1".
+3. Recompile your projects.
+
+And that's it!  Because your projects now target .NET Framework 4.6.1, you can use that version of .NET Framework as your base for porting code.
+
+## How to Multitarget with .NET Core
+
+Many libraries multitarget to have as wide of a reach as possible.  With .NET Core, multitargeting is a "first class citizen", meaning that you can easily generate platform-specific assemblies.
+
+Multitargeting is as simple as adding the correct Target Framework Moniker (TFM) to your `project.json` file, pulling in the correct dependecies (`dependences` for .NET Core and `frameworkAssemblies` for .NET Framework), and potentially using `#if` compile guards to conditionally compile source.
+
+For example, imagine you were building a library where you wanted to perform some networking, and you wanted that library to run on all .NET Framework versions, a Portable Class Library (PCL) Profile, and .NET Core.  For .NET Core and .NET Framework 4.5+ targets, you may use `System.Net.Http` and `async`/`await`.  However, for lower versions of .NET Framwork, those APIs aren't available.
```

Comment:
typo: .NET Framwork. 
Also: 
1. .net framework should be preceded by the article "the". 
2. shouldn't it be earlier versions of the framework?